SchoolTask

pavel-vel opened the pull request:
I'm sure this is far from the best way to get this done but honestly - I'm happy I figured out a way for it to work at all! 

Here is what the task was:

```
We are given a school. In the school there are classes of students. 
Each class has a set of teachers. Each teacher teaches a set of disciplines.
Students have name and unique class number. 
Classes have a unique text identifier. 
Teachers have name and title. 
Disciplines have name, number of lectures and number of exercises.

Define Java classes for the school (School, Class, Student, Teacher, Discipline). 
Keep the member fields private. 
Add constructors and accessor methods. 
Write a testing class to construct and print a sample school.
```

Looking forwards to your suggestions :)

p-avel committed:
This is a WIP, testing Commit and Push

p-avel committed:
Hopefully! reworked this according to OOP.
Replaced arrays w arraylists
Fixed things I'd missed last time (e.g. displaying Disciplines)

p-avel committed:
Hopefully! reworked this according to OOP.
Replaced arrays w arraylists
Fixed things I'd missed last time (e.g. displaying Disciplines)

kdaskalova-mlb commented on SchoolExample/Class.java:
```diff
@@ -1,14 +1,20 @@
 
-    public Class(String textId) {
-        this.textId = textId;
+    public Class(String textId, ArrayList<Student> uchenici, ArrayList<Teacher> uchiteli) {
```
Please, change list `uchenici` and list `uchiteli` to English.

kdaskalova-mlb commented on SchoolExample/Demo.java:
```diff
@@ -1,10 +1,11 @@
 
-        School tumba = new School("Tumba");
-
 //      Ученици
```
Comments should be in english.

kdaskalova-mlb commented on SchoolExample/School.java:
```diff
@@ -1,11 +1,15 @@
+
+    public School(String name, ArrayList klasove) {
         this.name = name;
+        this.klasove = klasove;
```
The filed 'klasove` should be in english.

p-avel committed:
Hopefully! reworked this according to OOP.
Replaced arrays w arraylists
Fixed things I'd missed last time (e.g. displaying Disciplines)
